extensions/divina.md

```diff
@@ -192,53 +182,64 @@ As an alternative, the manifest can also be added to a CBZ file at the same well
 
 * Support for the [Readium Web Publication Manifest](https://readium.org/webpub-manifest) with bitmap images in `readingOrder`
 * Support for [alternate resources](#3-alternate-resources)
-* Support for the rendition extension (TBD)
+* Support for [presentation hints](presentation.md)
 
 ### Level 1
 
 * Support for [guided navigation](#4-guided-navigation)
+* Support for transitions
 
 ### Level 2
 
 * TBD
 
-## Appendix B. Webtoons
-
-*This section is non-normative.*
+## Appendix B. Examples
 
-> **Note:** We're currently missing the ability to indicate how an image should fit on the screen. The current default is to fit both dimensions on screen, which would not work well with webtoons. In this non-normative appendix, we're currently using `fit` for this use case.
+*Example 5: A manga is a DiViNa where images are presented sequentially from right-to-left with a discontinuity between images that are not in the same spread*
 
-Webtoons are probably the most successful form of digital native visual narrative. Originally from South Korea, they're now becoming popular in Japan and France as well and should be covered by this profile.
-
-We could either consider that a webtoon is:
-
-- a single scrollable image, that should fit the width of the viewport
-- multiple images that can be continuously scrolled
-
-With the first approach, the `readingOrder` contains a single image:
 
 ```
+"metadata": {
+  "title": "Manga",
+  "identifier": "https://example.com/manga",
+  "readingProgression": "rtl",
+  "presentation": {
+    "fit": "contain",
+    "spread": "landscape"
```

Comment:
manga production is mostly in portrait mode.

Reply:
This is for spreads, which IMO is correct = display spreads only in landscape mode.

Reply:
you're right.